wiki/forms.py

```diff
@@ -124,6 +124,11 @@ class CreateRootForm(forms.Form):
         required=False, widget=getEditor().get_widget())  # @UndefinedVariable
 
 
+class MoveForm(forms.Form):
+
+    destination = forms.CharField(label=_('Destination'), initial='', help_text=_('Childrens are also moved. Be carefull, links are NOT updated !'))
```

Comment:
I'm a bit confused -- what's this field actually used for? It seems it's for a PK for URLPath? Should it perhaps be made a ModelChoiceField with widget=TextInput() ?